model/portableElement/PortableElementService.php

```diff
@@ -335,6 +335,25 @@ public function getPortableElementByClass($portableElementClass, Element $qtiIte
             }
         }
 
+        //remove base url
+        array_walk($portableElements, function(&$data){
```

Comment:
I see, would you have a preference ? Removing it would be "cleaner" for delivery assembly data but making a major version for this would means that instances using old minor one will not get the fix. So would leaving and deprecating it as your suggestion 2 be the better solution ?